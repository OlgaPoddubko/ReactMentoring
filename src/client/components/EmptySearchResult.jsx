import React from 'react';

const EmptySearchResult = ()=> (
    <div className="empty-search-results">No films found.
      <style jsx>{`
        .empty-search-results {
          max-width: 1130px;
          margin: 0 auto;
          padding: 60px 35px;
          text-align: center;
          font-weight: bold;
          font-size: 48px;
        }
      `}</style>
    </div>
);

export default EmptySearchResult;
