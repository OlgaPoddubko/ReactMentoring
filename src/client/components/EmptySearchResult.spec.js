import React from 'react';
import EmptySearchResult from './EmptySearchResult';

describe('render EmptySearchResult', () => {
  it('should render EmptySearchResult', () => {
    const wrapper = shallow(<EmptySearchResult />);
    expect(wrapper).toMatchSnapshot();
  });
});
