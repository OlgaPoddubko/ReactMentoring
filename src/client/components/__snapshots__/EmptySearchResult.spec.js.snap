// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render EmptySearchResult should render EmptySearchResult 1`] = `
<div
  className="empty-search-results"
>
  No films found.
  <style
    jsx={true}
  >
    
        .empty-search-results {
          max-width: 1130px;
          margin: 0 auto;
          padding: 60px 35px;
          text-align: center;
          font-weight: bold;
          font-size: 48px;
        }
      
  </style>
</div>
`;
